### ISSUE TITLE
Balancing Ball - questions about training process

### ISSUE THREAD

**Issue by NinaMaz** [Unity-Technologies/ml-agents#280]
Hello! 
I have completed the tutorial on Balancing Ball (using PPO), however, received quite unsatisfying results with the default parameters from the notebook. I have noticed that I receive very unstable results when training for 500000 steps: 
![ppo-3dball1](https://user-images.githubusercontent.com/14794151/35277698-a803b156-0058-11e8-9129-b8131efed00d.PNG)

When doing the quick training (50000 steps, bigger `buffer_size` and `batch_size`), it seems to go pretty well, apart from the `value_loss`, which is increasing instead of the opposite:

![ppo-3dball2](https://user-images.githubusercontent.com/14794151/35277756-db1abeae-0058-11e8-9efd-de002a2f6695.PNG)

Why do you think, even when training for such a long time, I can't reach a stable policy? I would be happy with the results given by my shorter training session, but the value_loss confuses me a lot. 

Maybe there is a way for me to receive better results by tuning the hyperparameters? If so, what would you suggest to do?

**Comment by awjuliani** [comment]
Hi @NinaMaz ,

I would not worry about value_loss, as it increasing simply means that your agent is discovering new things about the environment which the model hasn't accounted for yet. The main thing to focus on as a signal of the goodness of your agents is the cumulative reward. The reason there is the fluctuation you see is that there is a large amount of variance in the training signal. By increasing buffer size and batch size you decrease that variance, which leads to more stable training. 3DBall is actually a relatively simple problem to solve, so only a few tens of thousands of steps are required, more than that opens the possibility of forgetting what is learned by the agent.
